.eslintrc.js

```diff
@@ -60,9 +60,9 @@ module.exports = {
     'brace-style': ['error', '1tbs', { allowSingleLine: true }],
     'capitalized-comments': ['error', 'always', {
       line: {
-        // Ignore all lines that have less characters than 30 and all lines that
+        // Ignore all lines that have less characters than 20 and all lines that
         // start with something that looks like a variable name or code.
-        ignorePattern: '^.{0,30}$|^ [a-z]+ ?[0-9A-Z_.(/=:[#-]|^ std',
+        ignorePattern: '^.{0,20}$|^ [a-z]+ ?[0-9A-Z_.(/=:[#-]|^ std',
```

Comment:
I don't think you need the `^` at the start or the `^ ` instances later in the regexp [because ESLint will only match on the start of the comment string excluding whitespace](https://github.com/eslint/eslint/blob/78358a8f66e95c4fcc921f2497e8a5ec5f1537ec/lib/rules/capitalized-comments.js#L94). (If you do take them out, test it of course....)